src/components/Dashboard/TodoList.vue

```diff
@@ -310,6 +310,8 @@
               <v-btn
                 class="ma-1 affiliation-invitation-action-button"
                 color="primary"
+                :loading="isAuthorizeLoading"
+                :disabled="isAuthorizeLoading || isDoNotAuthorizeLoading"
```

Comment:
I think it's automatically disabled when it's loading. Try this without the first condition.

Reply:
Tried to only have the second condition. It doesn't automatically disabled the button when loading.

##### With only the second condition:
https://github.com/user-attachments/assets/c0462a47-2852-4f51-859d-f18def11ce12

##### With 2 conditions (disabled and loading):
https://github.com/user-attachments/assets/7695750e-61e7-4f38-9387-808994e2e0a9

Reply:
OK, thanks for trying this. I stand corrected :)

Resolving this.